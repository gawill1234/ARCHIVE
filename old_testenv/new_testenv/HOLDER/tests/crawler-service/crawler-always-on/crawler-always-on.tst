#!/bin/bash

#####################################################################

###
#   Global stuff
###
   
   source $TEST_ROOT/lib/global_setting.sh

   VCOLLECTION="crawler-always-on"
   DESCRIPTION="Refresh with concurrent queries"

###
###

source $TEST_ROOT/lib/lib_func.sh
source $TEST_ROOT/lib/lib_query.sh

#####################################################################
test_header $VCOLLECTION $DESCRIPTION

speed=$1
if [ "$speed" == "" ]; then
   speed="slow"
fi

echo "SPEED:  $speed"

if [ -f /testenv/samba_test_data/refresh-stress/stable/us_doi.pdf ]; then
   echo "/testenv is mounted.  We can continue"
else
   echo "/testenv directory is not mounted."
   echo "Create a local directory name /testenv.  Then, as root, do:"
   echo "mount -t nfs testbed5.test.vivisimo.com:/virtual2/data /testenv"
   echo "Then rerun the test"
   echo "refresh-stress: Test Failed"
   exit 1
fi

mkdir -p querywork

./crawler-always-on-oracle.sh $speed > querywork/oracle_output &
rsoracle=$!
echo "ORACLE:  $rsoracle"

./crawler-always-on-postgres.sh $speed > querywork/postgres_output &
rspostgres=$!
echo "POSTGRES:  $rspostgres"

./crawler-always-on-sharepoint.sh $speed > querywork/sharepoint_output &
rssharepoint=$!
echo "SHAREPOINT:  $rssharepoint"

./crawler-always-on-samba.sh $speed 1> querywork/samba_output 2> querywork/samba_output_err&
rssamba=$!
echo "SAMBA:  $rssamba"

sleep 2

echo "Wait for test parts to complete.  This will be awhile"
echo "Even if you said fast, this will still take 25 minutes"
echo "Waiting ..."
wait $rsoracle
oraexit=$?
wait $rspostgres
pgrexit=$?
wait $rssharepoint
shpexit=$?
wait $rssamba
smbexit=$?

finalstat=`expr $oraexit + $pgrexit + $shpexit + $smbexit`

if [ $finalstat -eq 0 ]; then
   delete_collection -C crawler-always-on-samba
   delete_collection -C crawler-always-on-oracle
   delete_collection -C crawler-always-on-postgres
   delete_collection -C crawler-always-on-sharepoint
   echo "crawler-always-on:  Test Passed"
   exit 0
else
   echo "oracle refresh:  exit($oraexit)"
   echo "postgres refresh:  exit($pgrexit)"
   echo "sharepoint refresh:  exit($shpexit)"
   echo "samba refresh:  exit($smbexit)"
   echo "crawler-always-on:  Test Failed"
   stop_indexing -C crawler-always-on-samba
   stop_indexing -C crawler-always-on-oracle
   stop_indexing -C crawler-always-on-postgres
   stop_indexing -C crawler-always-on-sharepoint
fi

exit 1
