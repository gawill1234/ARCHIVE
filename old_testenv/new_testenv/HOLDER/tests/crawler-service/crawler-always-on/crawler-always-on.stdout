TEST:  crawler-always-on
TEST DESCRIPTION:  Refresh with concurrent queries
PING testbed4.test.vivisimo.com (192.168.0.45) 56(84) bytes of data.
64 bytes from testbed4.test.vivisimo.com (192.168.0.45): icmp_seq=1 ttl=64 time=0.144 ms

--- testbed4.test.vivisimo.com ping statistics ---
1 packets transmitted, 1 received, 0% packet loss, time 0ms
rtt min/avg/max/mdev = 0.144/0.144/0.144/0.000 ms
Test begins ...
Query service is OK
COPY: /home/gaw/TESTENV/working/testenv/testrun/64bit/VTD_19374/VTD_11734/crawler-always-on/6-1.crawler-always-on-postgres.xml
TO  : /home/gaw/TESTENV/working/testenv/testrun/64bit/VTD_19374/VTD_11734/crawler-always-on/crawler-always-on-postgres.xml
COPY: /home/gaw/TESTENV/working/testenv/testrun/64bit/VTD_19374/VTD_11734/crawler-always-on/6-1.crawler-always-on-sharepoint.xml
TO  : /home/gaw/TESTENV/working/testenv/testrun/64bit/VTD_19374/VTD_11734/crawler-always-on/crawler-always-on-sharepoint.xml
COPY: /home/gaw/TESTENV/working/testenv/testrun/64bit/VTD_19374/VTD_11734/crawler-always-on/6-1.crawler-always-on-sharepoint.xml.stats
TO  : /home/gaw/TESTENV/working/testenv/testrun/64bit/VTD_19374/VTD_11734/crawler-always-on/crawler-always-on-sharepoint.xml.stats
COPY: /home/gaw/TESTENV/working/testenv/testrun/64bit/VTD_19374/VTD_11734/crawler-always-on/6-1.crawler-always-on-postgres.xml.stats
TO  : /home/gaw/TESTENV/working/testenv/testrun/64bit/VTD_19374/VTD_11734/crawler-always-on/crawler-always-on-postgres.xml.stats
COPY: /home/gaw/TESTENV/working/testenv/testrun/64bit/VTD_19374/VTD_11734/crawler-always-on/6-1.crawler-always-on-oracle.xml.stats
TO  : /home/gaw/TESTENV/working/testenv/testrun/64bit/VTD_19374/VTD_11734/crawler-always-on/crawler-always-on-oracle.xml.stats
COPY: /home/gaw/TESTENV/working/testenv/testrun/64bit/VTD_19374/VTD_11734/crawler-always-on/6-1.crawler-always-on-oracle.xml
TO  : /home/gaw/TESTENV/working/testenv/testrun/64bit/VTD_19374/VTD_11734/crawler-always-on/crawler-always-on-oracle.xml
SPEED:  slow
/testenv is mounted.  We can continue
ORACLE:  12214
POSTGRES:  12215
SHAREPOINT:  12221
SAMBA:  12224
Wait for test parts to complete.  This will be awhile
Even if you said fast, this will still take 25 minutes
Waiting ...
oracle refresh:  exit(3)
postgres refresh:  exit(0)
sharepoint refresh:  exit(0)
samba refresh:  exit(4)
crawler-always-on:  Test Failed
Stop indexing (admin) for  crawler-always-on-samba
Stop indexing (admin) for  crawler-always-on-oracle
Stop indexing (admin) for  crawler-always-on-postgres
Stop indexing (admin) for  crawler-always-on-sharepoint
